docs/detections/api/rules/rules-api-create-rule-default-exception-list.asciidoc

```diff
@@ -0,0 +1,105 @@
+[[exceptions-api-create-rule-default-exception-list]]
+=== Create default exception list for a rule
+
+Creates a default exception list for the rule you specify. Default exception lists can only be associated with a single rule.
```

Comment:
My wording isn't a great, but maybe something like:

Adds specified exception items to a rules default exception list - if no default exception list exists, one is created and associated with the rule. Default exception lists are meant to house exceptions associated with a single rule, as opposed to shared exception lists which can be associated to multiple rules.